File: practices/conditionals.py
# BB 7th conditionals

#homework_done = input("did you do your homework?").strip().capitolize()

#if homework_done == "no":
    #print("go do your homework")
#else:
    #print("Yes you can go")

grade =  int(input("What ")) 

if grade >= 90 and not grade > 100:
    print(f"you have {grade}% what is an A!")
elif grade >= 80:
    print(f"you have {grade}% that is a B")
else:
    print(f"you have a {grade}% that is a C or lower")

name = input("what is your name: ").strip().capitalize()

if name == "Larose":
    print("You are the teacher")
elif name == "Tia" or name == "Ashley":
    print("You are TA")
else:
    print(f"Hello {name}, you are student")

File: c_programming/notes/conditionals.c
// BB 7th Conditionals c

#include <stdio.h>
#include <string.h>

int main(void){
    int grade;
    char name[50];
    printf("What is your grade percent");
    scanf("%d", &grade);

     printf("What is your name");
    scanf("%s", &name);

        //printf("%d\n",strcmp(name, "Ms.LaRose"));
    if(strcmp(name, "Ms.LaRose") == 0){
        printf("You dont have a grade");
    }else if(grade >= 90){
        printf("you have an A\n");
    }else if(grade >= 80){
        printf("You have a B\n");
    }else if(grade >= 70){
        printf("You have a C\n");
    }else{
        printf("You are failing\n");
    }


}